python/tvm/micro/model_library_format.py

```diff
@@ -67,56 +69,75 @@ def generate_c_interface_header(
 EPHEMERAL_MODULE_TYPE_KEYS = ("metadata_module",)
 
 
-def _populate_codegen_dir(mod, codegen_dir: str, module_name: str = None):
+def _populate_codegen_dir(
+    mods: Union[
+        typing.List[executor_factory.ExecutorFactoryModule], typing.List[tvm.runtime.Module]
```

Comment:
I changed it to `build_module.OperatorModule`
I think Union of List is correct because the input could be Union of two different list, but not a list of two different type

Reply:
actually since we are passing modules like `build_module.OperatorModule` and `tvm.support.FrontendTestModule` I changed it back to tvm.runtime.Module